From 093451aa4e4e732a0862cd504c4065a367a559b4 Mon Sep 17 00:00:00 2001
From: Armin Kuster <akuster808@gmail.com>
Date: Fri, 12 Nov 2021 01:50:02 +0000
Subject: [PATCH] clamd not installing clamav-daemon.socket

Fixes:
__main__.SystemdUnitNotFoundError: (PosixPath('../security-build-image/1.0-r0/rootfs'), 'clamav-daemon.socket')
%post(clamav-daemon-0.104.0-r0.core2_64): waitpid(3587571) rc 3587571 status 100
warning: %post(clamav-daemon-0.104.0-r0.core2_64) scriptlet failed, exit status 1

Upstream-Status: Pending
Signed-off-by: Armin Kuster <akuster808@gmail.com>

---
 clamd/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/clamd/CMakeLists.txt b/clamd/CMakeLists.txt
index 9c3689f..f8170cc 100644
--- a/clamd/CMakeLists.txt
+++ b/clamd/CMakeLists.txt
@@ -54,4 +54,10 @@ if(SYSTEMD_FOUND)
     install(
         FILES ${CMAKE_CURRENT_BINARY_DIR}/clamav-daemon.service
         DESTINATION ${SYSTEMD_UNIT_DIR})
+    configure_file(
+        ${CMAKE_CURRENT_SOURCE_DIR}/clamav-daemon.socket.in
+        ${CMAKE_CURRENT_BINARY_DIR}/clamav-daemon.socket @ONLY)
+    install(
+        FILES ${CMAKE_CURRENT_BINARY_DIR}/clamav-daemon.socket
+        DESTINATION ${SYSTEMD_UNIT_DIR})
 endif()
